packages/design-system/src/components/tree/index.tsx

```diff
@@ -559,7 +559,7 @@ export function TreeItem<T>({
       ],
       // Need to expand the RAC ButtonContext to allow for additional slots
       [RACButtonContext, { slots: { [DEFAULT_SLOT]: {}, lines: {} } }],
-    ];
+    ] as TreeItemContexts;
```

Comment:
nit: is there a way to do this without typecasting it? ie: building it above this with populating all required params?

Reply:
cc @ArrayKnight since he did this change